config/routes.rb

```diff
@@ -234,6 +234,10 @@
         resources :courses, only: %i[index show]
         resources :users, only: %i[index show]
 
+        resources :participant_outcomes, only: %i[] do
+          member { get :resend }
```

Comment:
@cwrw I've followed ECF and made this a `GET` but it feels like this should be a `POST` since its modifying data? Thoughts?

Reply:
yea `POST` sounds better tbh 👍